lib/foreman_maintain/package_manager/dnf.rb

```diff
@@ -1,10 +1,44 @@
 module ForemanMaintain::PackageManager
   class Dnf < Yum
+    PROTECTOR_CONFIG_FILE = '/etc/yum/pluginconf.d/foreman-protector.conf'.freeze
+    PROTECTOR_WHITELIST_FILE = '/etc/yum/pluginconf.d/foreman-protector.whitelist'.freeze
+
     def clean_cache(assumeyes: false)
       dnf_action('clean', 'all', :assumeyes => assumeyes)
       super
     end
 
+    def dnf_plugin_dirs
```

Comment:
I am expecting python version will be newer in EL9 hence trying to catch all dirs which have child `dnf-plugin` dir